transactions.md

```diff
@@ -128,32 +128,31 @@ fileName:transaction.json
+## Transaction Event Types
+In the life cycle of a transaction, there are multiple transaction events. One transaction event occurs at authorization time. Another transaction event occurs when the transaction has cleared, meaning the funds were successfully moved. If a payment gets reimbursed, that results in a third event. These are distinct events, and Fidel API processes all of them, even if you're not registering webhooks to listen for all event types.
 
-**Clearing transactions**, also known as “settled transaction”, are processed when a payment transaction settles, usually happens 48 to 72 hours after a payment registers. The Fidel processes for clearing transactions are also triggering the `transaction.clearing` webhook events. The processes run daily at 12:00 UTC for Mastercard and multiple times per day for Visa and American Express. Only one transaction is sent per event.
+Fidel API currently supports three types of transactions: [authorization transactions](#authorization), [clearing transactions](#clearing) and [refund transactions](#refund).
 
-**Refund transactions** are processed when a payment is refunded, usually when a purchased item is returned, and the payment reverses. A refunded transaction triggers two webhook events, `transaction.clearing` and `transaction.refund`, with the `auth` property set to false. The amount on both events is negative. The Fidel API tries to identify the initial transaction for which the refund was issued, using `cardId`, `locationId`, `merchantId`, `amount` and `datetime`. If an associated initial transaction is identified, the webhook data contains the `originalTransactionId`. If no initial transaction is identified, the data comes in on both webhooks with a negative amount but no `originalTransactionId` property.
+## Authorization 
+Authorization transactions are processed when a purchase is made on a linked card at an auth-enabled location (either in-store or online). For example, when a customer makes a purchase at an auth-enabled location with their linked card, the `transaction.auth` webhook will be triggered and the transaction object will be sent to your specified URL in real time.
 
-The original transaction has a new `refundTransactionId` property, set to the `transactionId` of the refunded transaction. Updates on the original transaction will not trigger a webhook event.
+## Clearing
+Clearing transactions - also known as settled transactions - are processed when a payment transaction settles. This usually happens 48 to 72 hours after a payment is made. Clearing transactions trigger the `transaction.clearing` webhook event. The processes run daily at 12:00 UTC for Mastercard and multiple times per day for Visa. Only one transaction is sent per event.
```

Comment:
Mastercard runs twice a day
Visa are cleared as soon they are received.
We are missing Amex

Reply:
@krisztina-nagy how should we document this? Is it worth having this specified in the docs? It does not add much value to specify when our process runs, or if it is a cron job or an event-based approach. saying that it might take up to 72 hours probably is more than enough?

Reply:
I think the "up to 72 hours" is enough. Too much text on this subject would be distracting.

Reply:
removed. 👍